dev-client/src/components/inputs/TextInput.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {forwardRef} from 'react';
+import {TextInput as RNTextInput, StyleSheet} from 'react-native';
+import {
+  TextInput as RNPTextInput,
+  TextInputProps as RNPTextInputProps,
+} from 'react-native-paper';
+
+import {theme} from 'terraso-mobile-client/theme';
+
+export type TextInputProps = RNPTextInputProps;
+
+export const TextInput = forwardRef<RNTextInput, TextInputProps>(
```

Comment:
Issue: There is an error in the console, which we should avoid introducing:
> ERROR  Warning: forwardRef render functions accept exactly two parameters: props and ref. Did you forget to use the ref parameter?

I suspect you'll need to remove the forwardRef from here too to solve that.

Reply:
I've pulled the ref back in an will clean that up on a separate branch.

Reply:
Let's do that in another PR. I'll as @shrouxm for help.

Reply:
Looked into this and actually, never mind! Removing the ref will mess up the focus for the cursor on Site Notes.